dropwizard-parent-pom/pom.xml

```diff
@@ -144,67 +140,12 @@
         <scope>import</scope>
       </dependency>
 
-      <!-- Dependencies that Dropwizard uses. -->
-      <dependency>
```

Comment:
Most of these dependencies are already configured through importing the dropwizard-bom.